src/sensors/softfusion/softfusionsensor.h

```diff
@@ -76,7 +76,26 @@ class SoftFusionSensor : public Sensor {
 	bool detected() const {
 		const auto value
 			= m_sensor.m_RegisterInterface.readReg(SensorType::Regs::WhoAmI::reg);
-		if (SensorType::Regs::WhoAmI::value != value) {
+		if constexpr (requires { SensorType::Regs::WhoAmI::values.size(); }) {
+			for (auto possible : SensorType::Regs::WhoAmI::values) {
+				if (value == possible) {
+					return true;
+				}
+			}
```

Comment:
nit: could use std::find or std::any_of instead, though it's unclear if that would improve on readability